# 期望弯折角度
float32 kink_angle_target